Scripts/getLatestVersion.ps1

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<# 
+.Synopsis
+    Retrieve the latest version of the library
+.Description 
+    Retrieves the latest version specified in the Gradle.Properties file
+    Uses the retrieved values to update the enviornment variable VERSION_STRING
+#>
+
+.Parameter propertiesPath
```

Comment:
Needs to be within the comment

Reply:
good catch, addressed in https://github.com/microsoftgraph/msgraph-sdk-java-core/pull/57, I'll copy here